src/services/fraud-prover.service.ts

```diff
@@ -214,42 +210,40 @@ export class FraudProverService extends BaseService<FraudProverOptions> {
       await sleep(this.options.pollingInterval)
 
       try {
-        this.logger.info(`Looking for mismatched state roots...`)
+        this.logger.info('Looking for mismatched state roots...')
         const fraudulentStateRootIndex = await this._findNextFraudulentStateRoot()
 
         if (fraudulentStateRootIndex === undefined) {
-          this.logger.info(
-            `Didn't find any mismatched state roots. Trying again in ${Math.floor(
-              this.options.pollingInterval / 1000
-            )} seconds...`
-          )
+          this.logger.info('Did not find any mismatched state roots', {
+            nextAttemptInS: this.options.pollingInterval / 1000,
```

Comment:
What do you think about:
```
retry: ${this.options.pollingInterval / 1000}s,
```

So it looks like

```
retry: 2s
```

Reply:
Actually this flag seems to only apply to the timestamp of the log, not context data... in which case I think what you suggested is fine, but it may be slightly harder to aggregate that data.

Reply:
There exists tooling to parse such formatted strings: https://golang.org/pkg/time/#ParseDuration

Reply:
We could define a custom prettifier key. 

e.g.,
```ts
{
  customPrettifiers: {
    retryAgainS: prettifyQuery
  }
}
//...
const prettifyQuery = value => {
  return Math.floor(value / 1000) + 's'
}
```

Reply:
I've opened an issue for this: https://github.com/ethereum-optimism/optimism-ts-services/issues/34

Reply:
That seems like a good idea altho it makes the seconds indicator a bit redundant since its now in the key and in the value. It seems most useful when used across codebases. I think having the aggregations parse the values is more complex so I'm on board with your suggestion of putting the `S` in the key name